ema_workbench/em_framework/optimization.py

```diff
@@ -546,6 +745,32 @@ def __call__(self, optimizer):
             pass
 
 
+def epsilon_nondominated(results, epsilons, problem):
+    """Merge the list of results into a single set of
+    non dominated results using the provided epsilon values
+    Parameters
+    ----------
+    results : list of DataFrames
+    epsilons : epsilon values for each objective
+    problem : PlatypusProblem instance
+    Returns
+    -------
+    DataFrame
+    Notes
+    -----
+    this is a platypus based alternative to pareto.py (https://github.com/matthewjwoodruff/pareto.py)
+    """
+    if problem.nobjs != len(epsilons):
```

Comment:
Maybe we would rename this parameter to `n_objs` or just even `n_objectives`, that should be a lot clearer.

Reply:
not sure what you propose. The attribute `nobjs` on a PlatypusProblem instance is defined within platypus-opt and so outside the workbench.

Reply:
Ah right, then it's outside our control